build_support/jenkins.py

```diff
@@ -594,7 +594,10 @@ def write_summary(out_dir, completed_builds, ljen, failure=False):
         message = repo_set.repo(project).commit().message.splitlines()[0]
         # allow for special chars in message.
         git_log[project] = xml.sax.saxutils.quoteattr(message) 
-
+    rev_list = []
+    # rev_list is used for creating the retest build url in the summary
+    for (project, rev) in ljen._revspec._revisions.items():
```

Comment:
the 1 in there seems fishy

Reply:
also you could simplify this as:
```python
rev_list = ['{}={}'.format(p, r) for p, r in 1jen._revspec._revisions.items()]
```

Also, if this is a `dict` we should use iteritems instead of items (we have some container-ish classes that implement items but not iteritems)